tests/runtimes/local/models/models_test.py

```diff
@@ -262,3 +265,145 @@ def testModelsAPIKeyValidation_whenKeyIsInvalid_returnsFalse(
     api_key = models.APIKey.get_or_create()
 
     assert api_key.is_valid("invalid_key") is False
+
+
+def testAssetModels_whenCreateNetwork_assetCreated(
+    mocker: plugin.MockerFixture, db_engine_path: str
+) -> None:
+    """Ensure we correctly persist the network information."""
+    mocker.patch.object(models, "ENGINE_URL", db_engine_path)
+    models.Network.create(ips=["8.8.8.8", "42.42.42.42"])
+
+    with models.Database() as session:
+        assert session.query(models.Network).count() == 1
+        ips = json.loads(session.query(models.Network).all()[0].ips)
+        assert ips == ["8.8.8.8", "42.42.42.42"]
+
+
+def testAssetModels_whenCreateUrl_assetCreated(
+    mocker: plugin.MockerFixture, db_engine_path: str
+) -> None:
+    """Ensure we correctly persist the list of target URLs information."""
+    mocker.patch.object(models, "ENGINE_URL", db_engine_path)
+    models.Url.create(links=["https://example.com", "https://example42.com"])
```

Comment:
i think we need method here.

![image](https://github.com/Ostorlab/oxo/assets/144013278/c113928d-63d7-4a9f-b8ef-f667623bb4b1)

or they will be considered as domain names ?